src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -182,6 +184,10 @@ private ConfigProperties() {
     public static final String CACHE_ANON_CERT_CONTENT_MAX_ENTRIES =
         "candlepin.cache.anonymous.cert.content.max_entries";
 
+    // Paging
+    public static final String PAGING_SIZE = "candlepin.paging.size";
+    public static final String MAX_PAGING_SIZE = "candlepin.paging.max_size";
```

Comment:
can we make these a bit more clear? e.g.:
`candlepin.paging.default_page_size` and `candlepin.paging.max_page_size`?
That way the base namespace `candlepin.paging.` could be used in the future for other properties (e.g. `candlepin.paging.default_order` or `candlepin.paging.default_sort_field`

Reply:
That makes sense! I like `candlepin.paging.default_page_size` and `candlepin.paging.max_page_size` a lot more. Updated